Purchases/Networking/RCBackend.m

```diff
@@ -104,7 +104,9 @@ - (void)handlePurchaserInfoWithResponse:(nullable NSDictionary *)response
 - (void)handleResponse:(nullable NSDictionary *)response
             statusCode:(NSInteger)statusCode
                  error:(nullable NSError *)error
-            completion:(void (^)(NSError *_Nullable error))completion {
+            completion:(nullable void (^)(NSError * _Nullable error))completion {
```

Comment:
leaving these nullability changes in because `postAttributionData` calls this method with a nullable completion...